lib/index.js

```diff
@@ -31,7 +31,7 @@ exports.defaults = {
   branch: 'gh-pages',
   remote: 'origin',
   src: '**/*',
-  only: '.',
+  remove: '.',
```

Comment:
This default scares me.  I haven't found time to do a more thorough review, but I don't get why we would want to remove everything by default.

Reply:
Scratch that, I see that is the currently documented behavior.  Clearly I need to get my head back into this before understanding if this change does the right thing.

Reply:
I think removing all files default seems ok to me, as `options.add` is present for controlling this behaviour.
For a reference, please take a look  https://github.com/marketplace/actions/github-pages-action#%EF%B8%8F-keeping-existing-files